### ISSUE TITLE
Fatal error while resetting a projection: Error-Info: ERROR:  relation "projections" does not exis

### ISSUE THREAD

**Issue by jclaveau** [ecotoneframework/ecotone#8]
**Ecotone version(s) affected**: 1.10.4

**Description**  
On your project `php-ddd-cqrs-event-sourcing-symfony-ecotone` when I try to reset a projection as explained here https://docs.ecotone.tech/modelling/event-sourcing#resetting-the-projection, I get the following error

```shell
$.../php-ddd-cqrs-event-sourcing-symfony-ecotone$ docker-compose -f docker-compose.yml exec projections sh -l
$ bin/console ecotone:es:reset-projection last_prepared_tickets
PHP Warning:  Undefined array key "APP_DEBUG" in /data/app/bin/console on line 30
PHP Warning:  Undefined array key "APP_DEBUG" in /data/app/bin/console on line 38
[critical] Error thrown while running command "ecotone:es:reset-projection last_prepared_tickets". Message: "Error 42P01. 
Error-Info: ERROR:  relation "projections" does not exist
LINE 1: UPDATE "projections" SET status = $1 WHERE name = $2;
               ^"


In RuntimeException.php line 22:
                                                                 
  Error 42P01.                                                   
  Error-Info: ERROR:  relation "projections" does not exist      
  LINE 1: UPDATE "projections" SET status = $1 WHERE name = $2;  
                 ^                                               
                                                                 

ecotone:es:reset-projection <name>


```

**How to reproduce**  
clone the example
```shell
docker-compose -f docker-compose.yml exec projections sh -l
bin/console ecotone:es:reset-projection last_prepared_tickets
```

**Possible Solution**  
This error appears only when the db is empty. Triggering an event make it disappear. So it's more an false negative error than a bug.

**Context**  

Btw, thank you very much for your work!

**Comment by dgafka** [comment]
@jclaveau This is now fixed, please update your Ecotone Version to 1.10.5 and pull latest changes in Demo Project :)

**Comment by jclaveau** [comment]
Thank you!
